src/test/java/com/mxf/rest/ArticleDAOTest.java

```diff
@@ -0,0 +1,58 @@
+package com.mxf.rest;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.mxf.rest.dao.ArticleDAOService;
+import com.mxf.rest.entity.Article;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ComponentScan
+@EnableAutoConfiguration
+public class ArticleDAOTest {
+
+	@Autowired
+	private ArticleDAOService service;
+
+	@Test
+	public void testSelectAll() {
+		Article entity = new Article(0, "Title", "Label", "Test simple text", new Date(System.currentTimeMillis()), 2);
+		Integer id;
+		assertNotNull((id = service.create(entity)));
```

Comment:
On unit tests we have @Before and @After annotation. Pls check what it is and when it is better to use in your code